### PR TITLE
Edge case loading PR

### PR DESCRIPTION
**Pull request by pablonyx**
## Description
[Provide a brief description of the changes in this PR]

## How Has This Been Tested?
[Describe the tests you ran to verify your changes]

## Backporting (check the box to trigger backport action)
Note: You have to check that the action passes, otherwise resolve the conflicts manually and tag the patches.
- [ ] This PR should be backported (make sure to check that the backport attempt succeeds)

### CODE REVIEW

File: web/src/app/chat/ChatPage.tsx [hunk 1]
```diff
@@ -432,7 +432,9 @@ export function ChatPage({
 
       // if switching from one chat to another, then need to scroll again
       // if we're creating a brand new chat, then don't need to scroll
-      if (chatSessionIdRef.current !== null) {
+      // we must always set this to true once we animate a new chat session. Thi sis
```
Comment (Weves) [inline web/src/app/chat/ChatPage.tsx:435]: not sure I follow this comment 😅

File: web/src/app/chat/ChatPage.tsx [hunk 2]
```diff
@@ -2087,6 +2091,17 @@ export function ChatPage({
           </Modal>
         </div>
       )}
+      <div className="fixed pointer-events-none z-[100000000] inset-0 flex items-center justify-center text-default">
```
Comment (Weves) [inline web/src/app/chat/ChatPage.tsx:2094]: was this jsut for debugging?
